add docs for workloadPriority (#1170)
kubernetes-sigs/kueue: site/content/en/docs/tasks/_index.md

- Setup [Sequential Admission with Ready Pods](/docs/tasks/setup_sequential_admission).
- As a batch administrator, you can learn how to
  [monitor pending workloads](/docs/tasks/monitor_pending_workloads).

> - If you don't want to use pod's priority for workload, you can learn how to [run a Kueue managed Jobs with WorkloadPriority](/docs/tasks/run_job_with_workload_priority).

Reviewer: ```suggestion
-As a batch administrator, you can learn how to [run a Kueue managed Jobs with a custom WorkloadPriority](/docs/tasks/run_job_with_workload_priority).
```

This also might be more relevant for  batch users. And maybe administrators targeted section for the WorkloadPriority definition .

  Kueue supports MPIJob v2beta1, PyTorchJob, TFJob, XGBoostJob, and PaddleJob.
- [Run a Kueue managed KubeRay RayJob](/docs/tasks/run_rayjobs).
- [Submit Kueue jobs from Python](/docs/tasks/run_python_jobs).
- [Run a Kueue managed plain Pod](/docs/tasks/run_plain_pods).

> - [Run a Kueue managed Jobs with WorkloadPriority](/docs/tasks/run_job_with_workload_priority).

Reviewer 2: We should move this to `batch administrator` section.
